src/components/Navigation/Navigation.jsx

```diff
@@ -75,7 +78,7 @@ const Navigation = () => {
     >
       <List data-test="list-display">
         {links.map(({ name, endpoint }) => (
-          <ListItem button key={name}>
+          <ListItem button key={name} data-test="link-name">
```

Comment:
🔧 There's no need to add this, if we aren't using it.

Reply:
Addressed:  c73b28c7a4629611a85c7b5a81cd329fb4b36f5f